ui/src/components/song/SongListContainer.tsx

```diff
@@ -53,7 +75,9 @@ const SongListContainer: FC = (): ReactElement => {
                 ? filterData.themes.every((theme) => song.themes.includes(theme))
                 : true) &&
               (filterData.tempo
-                ? filterData.tempo.every((tempo) => song.tempo.includes(tempo))
+                ? filterData.tempo.every((tempo) =>
+                    song.tempo.map((t) => t.toLowerCase()).includes(tempo.toLowerCase())
+                  )
```

Comment:
for tempo and themes, I think its better if the filter logic is or instead of and